.pylintrc

```diff
@@ -455,7 +455,7 @@ min-similarity-lines=4
 [DESIGN]
 
 # Maximum number of arguments for function / method
-max-args=10
+max-args=20
```

Comment:
Mmm, what's the use case for that? Even 10 feels like a code smell for me, that the function/class does more than it supposed to.